readers/elasticsearch/src/main/resources/template.json

```diff
@@ -65,6 +65,18 @@
         }
       }
     },
+    "project_name": {
```

Comment:
please apply same config than the one for application_name field. The idea here is to have a project_name.keyword and workflow_name.keyword fields which are fields without split and normalisation 
"application_name": {
      "type": "text",
      "norms": false,
      "index_options": "positions",
      "fields": {
        "keyword": {
          "type": "keyword",
          "ignore_above": 1024
        }
      }
    },

Reply:
Done